src/target/opening-period-card/OpeningPeriodCard.scss

```diff
@@ -0,0 +1,50 @@
+@import '~hds-design-tokens/lib/all.scss';
+
+.opening-period-card {
+  font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
+  box-sizing: border-box;
+  border: 1px solid #e0e0e0;
+  background-color: #f9f9f9;
```

Comment:
Usually I prefer putting stuff to common once they are really shared by more than one component. This is what I referred when I talked about letting the software grow organically, instead of refactoring too early. It usually improves reading and you might not need to do the refactoring after all. 

That being said, it's not always so easy to judge when the refactoring should be made. Sometimes refactoring early lets developers more easily find potential common stuff that there is. For example I think abstracting http api to its own module early on was an ok choice, even it was only used in one place. Because it was so obvious that it was going there and there was more things coming to it. But with this OpeningPeriodCard and its colors, can we already tell that those colors will be used by other components? 🤔

Reply:
We discussed this and decided to put these colors into _variables.scss to have a common color palette for the app